test/models/Annotations.mo

```diff
@@ -0,0 +1,11 @@
+model A
+    Real x, y;
```

Comment:
Why not include an inline annotation here as well?

Reply:
I couldn't come up with one that makes sense. I just copied the large complicated annotation from a model that failed.

Reply:
fair enough. good for now.